.github/workflows/statictest.yml

```
@@ -1,43 +1,33 @@
-                  chmod -R +x $GITHUB_WORKSPACE/.tools/statictest
-                  mv $GITHUB_WORKSPACE/.tools/statictest /usr/local/bin/statictest
-            
-            - name: Run statictest
-              run: |
-                  go vet -vettool=$(which statictest) ./...
+  statictest:
+    runs-on: ubuntu-latest
+    container: golang:1.22
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)